grammar.js

```diff
@@ -321,18 +321,35 @@ module.exports = grammar({
     typeAnnotation: $ => seq(":", $.type),
 
     type: $ => choice(
-      "unknown",
-      "nothing",
-      $.stringConstant,
-      seq($.qualifiedIdentifier, optional($.typeArgumentList)),
-      seq("(", $.type, ")"),
-      prec(PREC.NULLABLE_TYPE, seq($.type, "?")),
-      seq($.type, "(", commaSep1($._expr), ")"),
-      prec.left(PREC.UNION_TYPE, seq($.type, "|", $.type)),
-      prec(PREC.DEFAULT_TYPE, seq("*", $.type)),
-      prec(PREC.FUN_TYPE, seq("(", commaSep($.type), ")", "->", $.type))
+      alias("unknown", $.unknownType),
+      alias("nothing", $.nothingType),
+      // TODO: module type
```

Comment:
You could just add `module` here and fix the conflict with the other PR.

Reply:
The plan was to merge the other PR first, then first this conflict which was always going to happen!